patterns/bottlerocket/README.md

```diff
@@ -0,0 +1,151 @@
+# Bottlerocket with Bottlerocket Update Operator
+
+This pattern demostrates how to deploy Amazon EKS Clusters using [Bottlerocket OS](https://aws.amazon.com/bottlerocket/) for Managed Node Groups (MNG) and [Karpenter](https://karpenter.sh/), using [Bottlerocket Update Operator (BRUPOP)](https://github.com/bottlerocket-os/bottlerocket-update-operator) to manage CVE patches automatically at the Node OS level. The BROPUP doesn't work with minor or major upgrades, just patch level.
+
```

Comment:
Gen we add a paragraph on the value of bottlerocket ? Why a customer should care and use bottlerocket for EKS, is it better than AL2 ? Is there some caveats to take into account ?

Reply:
I'm not sure if the blueprints pattern is the place for that. Since we're linking the official docs, customers can use that. But @askulkarni2 or @bryantbiggs can correct me if I'm wrong.